tests/Unit/Evolution/Systems/Cce/Test_ScriPlusInterpolationManager.cpp

```diff
@@ -44,22 +45,22 @@ void test_interpolate_quadratic() {
   //  f(u_bondi) = f_0 *(1.0 + a * u_bondi + b * u_bondi^2);
   //  where u_bondi is recorded with a small random deviation away from the
   //  actual time
+  // We use intervals of 0.1 for the sampling to give the interpolator an easier
+  // time
   for (size_t i = 0; i < data_points; ++i) {
     // this will give random times that are nonetheless guaranteed to be
     // monotonically increasing
     const DataVector time_vector = make_with_random_values<DataVector>(
         make_not_null(&generator), make_not_null(&time_dist), vector_size);
     interpolation_manager.insert_data(
-        time_vector + i,
+        time_vector + i * 0.1,
         random_vector *
-            (1.0 + linear_coefficient * (static_cast<double>(i) + time_vector) +
-             quadratic_coefficient *
-                 square(static_cast<double>(i) + time_vector)));
+            (1.0 + linear_coefficient * (i * 0.1 + time_vector) +
```

Comment:
For my understanding: instead making adjustments to `linear/quadratic_coefficient`, `i`, and `time_vector` ... could you not add a single adjustment here in front of the whole `linear_coefficient` term and similar for the quadratic term?

Reply:
There's actually a couple of adjustments here that have the same prefactor - I'm making the random time points closer together (with the 0.1 on the line 58 commented on) and also making the function vary more slowly (with the 0.1 multiplying the coefficients previously)